### PR TITLE
Update user-entity.md

### PR DESCRIPTION
**Pull request by vw98075**
Refine the one-to-one relationship with map id and cascade deletion as well as the newly required "builtInEntity".

### CODE REVIEW

**Review by DanielFran — APPROVED** [review]

**Comment by DanielFran** [comment]
@vw98075 Can you sign the CLA please?

File: pages/user-entity.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ entity ApplicationUser {
 }
 
 relationship OneToOne {
-  ApplicationUser{internalUser} to User
+  ApplicationUser{internalUser(login)} to @OnDelete("CASCADE") @MapId User with builtInEntity
```
Comment (mshima) [inline pages/user-entity.md:59]: I don't think `@MapId` exists.
```suggestion
  ApplicationUser{internalUser(login)} to @OnDelete("CASCADE") @Id User with builtInEntity
```

**Comment by renanfranca** [comment]
Congratulations @vw98075 for your first contribution 🎉👏

**Comment by vw98075** [comment]
@renanfranca Thanks. This isn't my first contribution to the project. Last year, I made some code corrections to the Vue front-end. However, similar errors existed in the React and Angular front-ends, so Matt assigned a JHipster team member to address those as well.

On a separate note, I reviewed the generated code today and noticed that the two annotations I made changes in the online document don't seem to have any effect. I'll create a ticket to investigate this further.

**Pull request merged**